index.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+	<html>
+	<head>
+		<link rel="stylesheet" type="text/css" href="css/styles.css">
+		<title>KANYE RESUME</title>
+	</head>
+
+	<body>
+		<nav></nav>
+		<nav></nav>
+			
+			<div class="header">
+				<h1>KANYE WEST</h1>
+
+				<ul>
+					<li>
+						<a href="quotes.html">QUOTES</a>
+					</li>
+					<li style="color: black"><b>||</b></li>
```

Comment:
:hammer: The `<b>` tag is deprecated, we don't use it anymore, although it works as you see. Also, we want to avoid inline styles here like you have on the `<li>`. Target both with a class. Something like this would have been preferred:
```html
<li class="some-class">||</li>
```
And the css would be:
```css
.some-class {
  color: black;
  font-weight: bold;
}
```